fuzzy_logic/fuzzy_operations.py.DISABLED.txt

```diff
@@ -28,7 +28,7 @@ if __name__ == "__main__":
     union = fuzz.fuzzy_or(X, young, X, middle_aged)[1]
     # 2. Intersection = min(µA(x), µB(x))
     intersection = fuzz.fuzzy_and(X, young, X, middle_aged)[1]
-    # 3. Complement (A) = (1- min(µA(x))
+    # 3. Complement (A) = (1- min(µA(x)))
```

Comment:
```suggestion
    # 3. Complement (A) = (1 - min(µA(x)))
```
While we're fixing typos, there should be an extra space before this subtraction sign